ddtrace/internal/compat.py

```diff
@@ -226,3 +226,35 @@ def get_connection_response(
     CONTEXTVARS_IS_AVAILABLE = False
 else:
     CONTEXTVARS_IS_AVAILABLE = True
+
+
+# Source: https://github.com/facelessuser/pep562
```

Comment:
We need to include their license as well, we might want to just add under `vendor/pep562.py` or something?

Reply:
What's the problem with adding the dependency?

Reply:
We could install the backport dependency like we do `enum34`.

Reply:
👍🏻